tron/core/actionrun.py

```diff
@@ -1134,6 +1135,8 @@ def handle_action_command_state_change(self, action_command, event, event_data=N
 class KubernetesActionRun(ActionRun, Observer):
     """An ActionRun that executes the command on a Kubernetes cluster."""
 
+    start_schedule_jobs = True
```

Comment:
I'm a little iffy on adding a boolean here - can we maybe add something to where we initialize the task_proc client instead?